bin/compile

```diff
@@ -173,6 +192,8 @@ summarize_build() {
   if $NODE_VERBOSE; then
     list_dependencies "$BUILD_DIR"
   fi
+
+  mmeasure 'modules.size' "$(measure_size)"
```

Comment:
I intentionally use ' when there should be no interpolation and " when there should be (since ' is safer).